_includes/header.html

```diff
@@ -51,24 +51,6 @@
 {% endif %}
 
 
-<!-- Header Javascript and CSS for animation-->
-<script src="/assets/js/vendor/jquery-1.11.0.min.js"></script>
-<script src="/assets/js/components.js"></script>
-<script src="/assets/js/vendor/modernizr.custom.js"></script>
-<script src="/assets/js/vendor/wow.min.js"></script>
```

Comment:
Should these files also be removed if we're not using vendored versions of them anymore